cs-powershell/src/main/java/io/cloudslang/content/actions/PowerShellScriptAction.java

```diff
@@ -140,31 +167,16 @@
                     .withTrustKeystore(trustKeystore)
                     .withTrustPassword(trustPassword)
                     .withScript(script)
+                    .withModules(modules)
                     .withWinrmLocale(winrmLocale)
                     .withOperationTimeout(operationTimeout)
                     .build();
 
-            resultMap = wsManRemoteShellService.runCommand(wsManRequestInputs);
+            Map<String, String> resultMap = wsManRemoteShellService.runCommand(wsManRequestInputs);
```

Comment:
`wsManRemoteShellService.runCommand(wsManRequestInputs); `
should not return a map with all the outputs populated, it should only return the minimum amount of information, in this method you should populate the output map